coderedcms/forms.py

```diff
@@ -145,7 +146,8 @@ class CoderedFormField(AbstractFormField):
     class Meta:
         abstract = True
 
-    field_type = models.CharField(verbose_name=_('field type'), max_length=16, choices=FORM_FIELD_CHOICES, blank=True)
+    field_type = models.CharField(verbose_name=_('field type'),
+                                  max_length=16, choices=FORM_FIELD_CHOICES, blank=True)
```

Comment:
This indentation looks wrong. Whenever declaring model fields / form fields, use this format:
```python
field = models.CharField(
    name = ...,
    attr = ...,
)